### PR TITLE
NXP-26227: fix DocumentModelCSVWriter when property is not found

### PR DESCRIPTION
**Pull request by pierre-gautier**
https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-pierregautier/213/

### CODE REVIEW

File: nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestSetPropertiesAction.java [hunk 1]
```diff
@@ -118,8 +118,12 @@ protected void testSetProperties(String username) throws Exception {
         for (DocumentModel child : session.query(nxql)) {
             assertEquals(title, child.getTitle());
             assertEquals(description, child.getPropertyValue("dc:description"));
-            assertEquals(foo, child.getPropertyValue("cpx:complex/foo"));
-            assertEquals(bar, child.getPropertyValue("cpx:complex/bar"));
+            try {
+                assertEquals(foo, child.getPropertyValue("cpx:complex/foo"));
+                assertEquals(bar, child.getPropertyValue("cpx:complex/bar"));
```
Comment (kevinleturc) [inline nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestSetPropertiesAction.java]: If I understand correctly, `try/catch` is used to not fail the test when child is a File or a Note.
You should check the document type to do these assertions.

File: nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java [hunk 2]
```diff
@@ -113,31 +113,28 @@ public void setWriteListener(WriteListener writeListener) {
...
 
     @Test
     public void testSimpleWithFileZipped() throws Exception {
-        BulkCommand command = createCommand(false, true);
-        testCsvExport(command);
+        testCsvExport(createCommand(false, true).build());
     }
 
-    public void testCsvExport(BulkCommand command) throws Exception {
+    protected void testCsvExport(BulkCommand command) throws Exception {
```
Comment (kevinleturc) [inline nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java]: Does it make sense to make parameter `BulkCommand.Builder` of this method and do the `build` inside ?
Also `createCommand` is a bit weird regarding its return type.

Comment (pierre-gautier) [reply nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java]: Giving a builder is not possible for testSimpleWithMultipleBuckets()
Renamed the method to createBuilder()

**Comment by pierre-gautier** [comment]
Updated
T&P https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-pierregautier/214/
first one failed on merge conflicts

**Comment by kevinleturc** [comment]
👍

**Comment by pierre-gautier** [comment]
👍

**Pull request merged**